docs/src/ScrollableContainer.doc.js

```diff
@@ -591,4 +600,19 @@ function ScrollableContainerExample() {
   </MainSection>,
 );
 
+card(
+  <MainSection
+    name="Related"
+    description={`
+[Modal](/Modal) / [Sheet](/Sheet)
+
+Modal and Sheet components have built ScrollableContainer internally to support the correct positioning of children on scroll as well as to keep the header and footer shadows when scrolling the content. Do not pass children to Modal and Sheet wrapped in ScrollableContainer so that they don’t loose the scrolling shadows.
```

Comment:
"Modal and Sheet come with ScrollableContainer built-in, so any anchored components used in their `children` tree should work out-of-the-box. Passing an additional ScrollableContainer will break the existing styling on scroll."

Reply:
DONE